CITATION.cff

```diff
@@ -0,0 +1,29 @@
+cff-version: 1.2.0
+title: Inscopix CNMF-E
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Bruno
+    family-names: Boivin
+    affiliation: Inscopix
```

Comment:
It might be more appropriate to use Inscopix Inc. here. I would double check with Shay.